src/rules/function-url-quotes/index.js

```diff
@@ -54,6 +63,9 @@ export default function (expectation) {
     }
 
     function checkArgs(args, node, index, functionName) {
+      const argumentIsEmpty = args === ""
```

Comment:
The `except` optional secondary option *reverses* the primary option. 

I don't think the following `"never"` config is accounted for:

`"function-url-quotes": "never", { except: ["empty"] }`

Your conditionally shouldn't return early and should [look like this](https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/index.js#L63-L67)